tooling/minimize/tests/pass/union.rs

```diff
@@ -20,4 +60,23 @@ fn main() {
     }
 
     let _y = B { f2: 0 };
+
+    let u = EnumUnion { data: Some(42) };
+    assert!(test_enum_union(u) == test_enum_union(u));
+
+    let u = StructUnion { data: TestStruct(12, 1200) };
+    let v1 = test_struct_union(u);
+    let v2 = test_struct_union(u);
```

Comment:
You're still calling all these functions twice, what's the point of that? Instead please just call it once and check that the result matches the original input.